docs/source/custom_resolvers.rst

```diff
@@ -121,14 +121,14 @@ the inputs themselves:
 
 Custom interpolations can also receive the following special parameters:
 
-- ``_parent_``: the parent node of an interpolation.
+- ``_parent_``: The parent node of an interpolation.
 - ``_root_``: The config root.
 
 This can be achieved by adding the special parameters to the resolver signature.
-Note that special parameters must be defined as named keywords (after the `*`).
+Note that special parameters must be defined as named keywords (after the ``*``).
 
-In the example below, we use ``_parent_`` to implement a sum function that defaults to 0 if the node does not exist.
-(In contrast to the sum we defined earlier where accessing an invalid key, e.g. ``"a_plus_z": ${sum:${a}, ${z}}`` would result in an error).
+In the example below, we use ``_parent_`` to implement a sum function that defaults to ``0`` if the node does not exist.
+(In contrast to the sum we defined earlier where accessing an invalid key, e.g. ``"a_plus_z": ${sum:${a}, ${z}}``, would result in an error.)
```

Comment:
First time I see period inside parentheses :).

Reply:
Changed to something more conventional in https://github.com/omry/omegaconf/pull/706/commits/ddecf6af4253e2b3685ee2d54af358bcf3e8df47.